interface.go

```diff
@@ -80,6 +80,12 @@ type Config struct {
 	// If not set, it verifies that the address matches, and that the STK was issued within the last 24 hours
 	// This option is only valid for the server.
 	AcceptSTK func(clientAddr net.Addr, stk *STK) bool
+	// MaxReceiveStreamFlowControlWindowServer is the maximum stream-level flow control window for receiving data
+	// If this value is zero, it will default to 1 MB for the server and 6 MB for the client
+	MaxReceiveStreamFlowControlWindow protocol.ByteCount
+	// MaxReceiveConnectionFlowControlWindowServer is the connection-level flow control window for receiving data
```

Comment:
Nit: Remove the "Server" from the comment, and add a dot at the end of the lines (godoc will put them in one paragraph).